roles/neutron-common/templates/etc/neutron/rootwrap.conf

```diff
@@ -12,9 +12,9 @@ filters_path=/etc/neutron/rootwrap.d,/usr/share/neutron/rootwrap,/etc/neutron/ro
 # If not specified, defaults to system PATH environment variable.
 # These directories MUST all be only writeable by root !
 {% if openstack_install_method == "package" -%}
-exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin,{{ 'neutron'|ursula_package_path(openstack_package_version) }}/bin
+exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin,/usr/local/bin,{{ 'neutron'|ursula_package_path(openstack_package_version) }}/bin
 {% else -%}
-exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin
+exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin,/usr/local/bin
```

Comment:
Leave /usr/local/bin, take out the templated openstack-source stuff, and take out the { 'neutron'|ursula_package_path(openstack_package_version) }} bits from the exec_dirs for package based install. In fact, that whole if statement can go away, it should be just a single set of paths.